snapcraft/internal/lxd/_containerbuild.py

```diff
@@ -76,6 +79,86 @@ def __init__(self, *, output, source, project_options,
             os.path.join('~', 'snap', 'lxd', 'common'))
         os.makedirs(self._lxd_common_dir, exist_ok=True)
 
+    def _setup_multipass_remote(self):
+        try:
+            containers = subprocess.check_output([
+                'multipass', 'list']).decode()
+            # Trailing space to disambiguate names starting with snapcraft
+            if 'snapcraft ' not in containers:
+                subprocess.check_call([
+                    'multipass', 'launch', '--name=snapcraft', '--disk', '10G',
+                    ])
+            subprocess.check_call([
+                'multipass', 'start', 'snapcraft'])
+            self._setup_lxd()
+        except FileNotFoundError:
+            raise ContainerError(
```

Comment:
Perhaps it's time to introduce a `MultipassError` and individual exceptions for these?